infra/src/main/java/com/redhat/service/smartevents/infra/models/gateways/Gateway.java

```diff
@@ -36,9 +37,14 @@ public class Gateway {
 
     @JsonProperty("parameters")
     // ObjectNode is not rendered properly by swagger
-    @Schema(implementation = Object.class, required = true)
+    @Schema(hidden = true)
     private ObjectNode parameters;
 
+    @JsonIgnore
+    // Workaround to generate the openapi.yaml file with a Map<String, Object> for the parameters. TODO: https://issues.redhat.com/browse/MGDOBR-788
+    @Schema(name = "parameters", required = true)
+    private Map<String, Object> justforopenapi;
```

Comment:
@r00ta after some discussion with @kelvah , we agreed changes in `Gateway.java` are not needed for now. Culd you please revert them?

Reply:
I'll close this PR then